integration_tests/src/main/python/parquet_write_test.py

```diff
@@ -33,16 +34,21 @@
               'spark.sql.legacy.parquet.int96RebaseModeInWrite': 'CORRECTED'}
 
 parquet_write_gens_list = [
-        [byte_gen, short_gen, int_gen, long_gen, float_gen, double_gen,
-            string_gen, boolean_gen, date_gen, timestamp_gen]]
+    [byte_gen, short_gen, int_gen, long_gen, float_gen, double_gen,
+     string_gen, boolean_gen, date_gen, timestamp_gen],
+    pytest.param([byte_gen, short_gen, int_gen, long_gen, float_gen, double_gen,
+                  string_gen, boolean_gen, date_gen, timestamp_gen, decimal_gen_default,
+                  decimal_gen_scale_precision, decimal_gen_same_scale_precision, decimal_gen_64bit],
+                 marks=pytest.mark.allow_non_gpu("CoalesceExec"))]
 
 parquet_ts_write_options = ['INT96', 'TIMESTAMP_MICROS', 'TIMESTAMP_MILLIS']
 
 @pytest.mark.parametrize('parquet_gens', parquet_write_gens_list, ids=idfn)
 @pytest.mark.parametrize('reader_confs', reader_opt_confs)
 @pytest.mark.parametrize('v1_enabled_list', ["", "parquet"])
 @pytest.mark.parametrize('ts_type', parquet_ts_write_options)
-def test_write_round_trip(spark_tmp_path, parquet_gens, v1_enabled_list, ts_type, reader_confs):
+def test_parquet_write_round_trip(spark_tmp_path, parquet_gens, v1_enabled_list, ts_type,
```

Comment:
do we really need parquet in the name when its in parquet_write_test?  just make it wrap inputs

Reply:
Changed it. I made the change to be able to execute the test by name otherwise it would run the orc test as well.

Reply:
ok that is fine. you can also specify the test file. src/main/python/parquet_write_test.py -k test_write_round_trip

Reply:
Thanks, that's really helpful for future.